internal/checks/peers.go

```
@@ -106,6 +106,8 @@ func (c *PeerCheck) runCheck() {
 }
 
 func (c *PeerCheck) IsPassing() bool {
+	// TODO(polsar): This method is unused. Instead, the decision whether this check is passing is made here:
+	//  https://github.com/satsuma-xyz/node-gateway/blob/b7f20aa2ad97f53772e9fa1565a300be7c0fff78/internal/route/node_filter.go#L61
```

If this method is unused then I'd remove it (or fail hard).